pkgs/development/python-modules/mkdocs-redirects/default.nix

```diff
@@ -1,29 +1,39 @@
 { lib
-, callPackage
 , buildPythonApplication
 , fetchFromGitHub
 , mkdocs
+, pytestCheckHook
+, pythonOlder
 }:
 
 buildPythonApplication rec {
   pname = "mkdocs-redirects";
-  version = "1.0.4";
+  version = "1.0.6";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "mkdocs";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-hdMfl8j+kZzePkSd/bBHKuVXAVA1sAt7DvPZj9x5i0c=";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-FAkuQ6A0ONx6H/q7H8NUk/epLpqfrQzFiyv+iM6yq2M=";
   };
 
   propagatedBuildInputs = [
     mkdocs
   ];
 
-  pythonImportsCheck = [ "mkdocs" ];
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "mkdocs"
```

Comment:
it feels like this should be `"mkdocs_redirects`" because that's what this module exports.